src/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -721,33 +721,125 @@ public String getID() {
+    
+    /** shortcut to {@link CommonInfo#getFiltersMaybeNull()} 
+     *  @return a <b>modifiable</b> Set of filters. Can be  null. All changes in this set will be reflected in the CommonInfo 
+     */
     public Set<String> getFiltersMaybeNull()    { return commonInfo.getFiltersMaybeNull(); }
+    
+    /** shortcut to {@link CommonInfo#getFilters()} 
+     * @return an unmodifiable Set of filters. Can be empty by never null
+     */
     public Set<String> getFilters()             { return commonInfo.getFilters(); }
+    
+    /** shortcut to {@link CommonInfo#isFiltered()}
+     * @return true if any filter been defined  <code>(!filters.isEmpty())
+     */
     public boolean isFiltered()                 { return commonInfo.isFiltered(); }
+    
+    /** shortcut to {@link CommonInfo#isNotFiltered()} */
     public boolean isNotFiltered()              { return commonInfo.isNotFiltered(); }
+    
+    /** shortcut to {@link CommonInfo#filtersWereApplied()}
+     *  @return true if no filter has been defined  <code>(getFiltersMaybeNull()==null)</code>
+     */
     public boolean filtersWereApplied()         { return commonInfo.filtersWereApplied(); }
+    
+    /** shortcut to {@link CommonInfo#hasLog10PError()}
+     * @return true if log10-based error estimate has been set 
+     */
     public boolean hasLog10PError()             { return commonInfo.hasLog10PError(); }
+    
+    /** shortcut to {@link CommonInfo#getLog10PError()}
+     * @return the -1 * log10-based error estimate
+     */
     public double getLog10PError()              { return commonInfo.getLog10PError(); }
+    
+    /** shortcut to {@link CommonInfo#getPhredScaledQual}
+     *  @return double - Phred scaled quality score
+     */
     public double getPhredScaledQual()          { return commonInfo.getPhredScaledQual(); }
-
+    
+    /** shortcut to {@link CommonInfo#getAttributes()}
+     * @return an unmodifiable Map of attributes
+     */
     public Map<String, Object>  getAttributes() { return commonInfo.getAttributes(); }
+    
+    /** shortcut to {@link CommonInfo#hasAttribute(String)}
+     * @return true if the key is present
+     */
     public boolean hasAttribute(String key)     { return commonInfo.hasAttribute(key); }
+    
+    /** shortcut to {@link CommonInfo#getAttribute(String)}
+     * @return the attribute value for the given key (or null if not set)
+     */
     public Object getAttribute(String key)      { return commonInfo.getAttribute(key); }
-
+    
+    /** shortcut to {@link CommonInfo#getAttribute(String, Object)}
+     * @return the attribute value for the given key (or defaultValue if not set)
+     */
     public Object getAttribute(String key, Object defaultValue) {
         return commonInfo.getAttribute(key, defaultValue);
     }
-
+    
+    /** shortcut to {@link CommonInfo#getAttributeAsString(String, String)}
+     * return an attribute as a String.
```

Comment:
You may want to use the javadoc `@see` tag in these methods to reference the appropriate docs in `CommonInfo`